### PR TITLE
Try to fix build with upstream MNE changes

### PR DESCRIPTION
**Pull request by hoechenberger**
### Before merging …

- [ ] Changelog has been updated (`docs/source/changes.md`)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -9,6 +9,7 @@ scikit-learn
 pandas
 json_tricks
 h5py
+pymatreader
```
Comment (agramfort) [inline requirements.txt:12]: should not be necessary

https://github.com/mne-tools/mne-python/blob/main/requirements_hdf5.txt

Comment (hoechenberger) [reply requirements.txt:12]: Oh, good catch

Comment (hoechenberger) [reply requirements.txt:12]:
```suggestion
```

**Comment by hoechenberger** [comment]
@agramfort `autoreject` tries to make use of `mne.externals`, hence doesn't work with MNE `main`

**Review by agramfort — APPROVED** [review]

**Comment by hoechenberger** [comment]
See https://github.com/autoreject/autoreject/issues/258

Until this has been resolved, our test suite here will partially fail

**Comment by hoechenberger** [comment]
I'm merging this now anyway 😅

**Comment by agramfort** [comment]
see https://github.com/autoreject/autoreject/pull/259

**Comment by hoechenberger** [comment]
Great!!

**Pull request merged**